ObjcVoiceQuickstart/ViewController.m

```diff
@@ -45,6 +45,9 @@ @interface ViewController () <PKPushRegistryDelegate, TVONotificationDelegate, T
 @property (weak, nonatomic) IBOutlet UISwitch *muteSwitch;
 @property (weak, nonatomic) IBOutlet UISwitch *speakerSwitch;
 
+@property (nonatomic, assign) BOOL answerOnBridgeEnabled;
```

Comment:
Do you think we should set this to connectOptions ?

Reply:
Wondering readers will get confused that by setting this flag answerOnBridge should get enabled for the call?  Thinking what would be the best here. going back to the original suggestion you had  `playLocalRingback` vs `answerOnBridgeEnabled` ? 🤔 

May be rename this to `playLocalRingback` and add comment explaining by default this flag is false and ringback tone is played by hURL by default. Enable this flag when answerOnBridge is set to true to play the ring back tone from a local file ?